lib/const.rb

```diff
@@ -1,21 +1,20 @@
 # Defines common constants used in different parts of crew
 
-CREW_VERSION = '1.16.9'
+CREW_VERSION = '1.16.10'
 
-ARCH_ACTUAL = `uname -m`.strip
+ARCH_ACTUAL = `uname -m`.chomp
 # This helps with virtualized builds on aarch64 machines
 # which report armv8l when linux32 is run.
-ARCH = if ARCH_ACTUAL == 'armv8l' then 'armv7l' else ARCH_ACTUAL end
+ARCH = ( ARCH_ACTUAL == 'armv8l' ) ? 'armv7l' : ARCH_ACTUAL
 
 # Allow for edge case of i686 install on a x86_64 host before linux32 is
 # downloaded, e.g. in a docker container.
-ARCH_LIB = if ARCH == 'x86_64' and Dir.exist?('/lib64') then 'lib64' else 'lib' end
+ARCH_LIB = ( ARCH == 'x86_64' and Dir.exist?('/lib64') ) ? 'lib64' : 'lib'
 
 # Glibc version can be found from the output of libc.so.6
-@libcvertokens=  %x[/#{ARCH_LIB}/libc.so.6].lines.first.chomp.split(/[\s]/)
-LIBC_VERSION = @libcvertokens[@libcvertokens.find_index("version") + 1].sub!(/[[:punct:]]?$/,'')
+LIBC_VERSION = %x[/#{ARCH_LIB}/libc.so.6][/version ([\d.]+)/, 1]
 
-if ENV['CREW_PREFIX'] and ENV['CREW_PREFIX'] != '' and ENV['CREW_PREFIX'] != '/usr/local'
+unless ENV['CREW_PREFIX'].to_s.empty? or ENV['CREW_PREFIX'] == '/usr/local'
```

Comment:
Does this throw an error if the environment variable `CREW_PREFIX` is not set?

Reply:
The `to_s` function handles it if it is not set (from nil to an empty string if not set)